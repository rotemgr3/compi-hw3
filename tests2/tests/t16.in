//piazza 84 - should be ambiguousCall +
override int foo(int x, int y) {int booga = 5;}
override bool foo(byte x, int y) {int booga = 5;}
void main() {
	int x = foo(5b,10);
}
