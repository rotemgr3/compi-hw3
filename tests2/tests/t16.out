---end scope---
x INT -1
y INT -2
booga INT 0
---end scope---
x BYTE -1
y INT -2
booga INT 0
line 5: ambiguous call to overloaded function foo
